### ISSUE TITLE
Necesito Bajar version de python

### ISSUE THREAD

**Issue by ManuelKastE** [IIC2233/Syllabus#37]
### Prerrequisitos
(Marcar colocando una `X` entre los corchetes los ítems que ya hiciste, así: `"[X]"`)

* [X ] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues).

### Problema

Holaa, no se como puedo bajar la version de python, actualmente tengo la version 3.12.1 pero en la wiki nos decian que necesitabamos la version 3.11.X. **Como puedo bajar la version de python en mi computador?**

Gracias!

**Comment by jorgee02** [comment]
Hola! Una de las formas más directas sería desinstalarlo completamente y luego ver en descargas la versión que necesites. Aquí está la de windows, por ejemplo.

https://www.python.org/downloads/windows/

**Comment by ManuelKastE** [comment]
Bueena bacan gracias, y como lo desinstalo?

**Comment by jorgee02** [comment]
Ahí dependerá de tu equipo y sistema operativo :D

**Comment by ManuelKastE** [comment]
tengo un macbook, pero pregunto porque me da miedo en vola queda la caga si llego y desinstalo python jajajajaja
o no pasa nada?

**Comment by jorgee02** [comment]
No, si lo haces normalmente y sigues las instrucciones que te aparezcan en pantalla, todo bien.

**Comment by ManuelKastE** [comment]
vale vale gracias
